### ISSUE TITLE
Semicolons in strings cause error (Postgres)

### ISSUE THREAD

**Issue by keithlayne** [Bajdzis/vscode-database#58]
Even a simple query like this:

```sql
select ';';
```

will fail with the following error:

```
PostgreSQL Error: error: unterminated quoted string at or near "'"
at Connection.parseE(/Users/keith/.vscode/extensions/bajdzis.vscode-database-1.5.2/node_modules/pg/lib/connection.js:567:11)
at Connection.parseMessage (/Users/keith/.vscode/extensions/bajdzis.vscode-database-1.5.2/node_modules/pg/lib/connection.js:391:17)
at Socket.<anonymous> (/Users/keith/.vscode/extensions/bajdzis.vscode-database-1.5.2/node_modules/pg/lib/connection.js:129:22)
at emitOne (events.js:116:13)
at Socket.emit (events.js:211:7)
at addChunk (_stream_readable.js:263:12)
at readableAddChunk (_stream_readable.js:250:11)
at Socket.Readable.push (_stream_readable.js:208:10)
at TCP.onread (net.js:594:20)
```

I tried this query with node-postgres and it worked fine, so I'm not sure what's happening here.

**Comment by Bajdzis** [comment]
I always split all queries by `;` before execute. 

related to #55

**Comment by Bajdzis** [comment]
Fixed in 2.2.0 version
